### PR TITLE
Adding Solr link & description

### PR DESCRIPTION
**Pull request by tpayet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -751,7 +751,7 @@ Synonyms: autocomplete, search as you type, suggestions
 * [Algolia](https://www.algolia.com/)
 * Vespa
 * Elastic
-* Solr
+* [Solr](https://solr.apache.org/) - OSS Elasticsearch alternative
```
Comment (epugh) [inline README.md:754]: I don't know that I would refer to Solr as an "Elasticseach alternative"....   The term "alternative" is fairly ambgious, and OpenSearch is actually a much closer alternative.   I'd rather hear about what makes it unique... 

The http://solr.apache.org website describes it as:

> Solr is the blazing-fast, open source, multi-modal search platform built on the full-text, vector, and geospatial search capabilities of Apache Lucene

Comment (tpayet) [reply README.md:754]: Gotcha 👍🏽  Do you want me to write anything else, or do you want only the link for the PR?

Comment (epugh) [reply README.md:754]: it's maybe a bit wordy, but could literally say "Solr is the blazing-fast, open source, multi-modal search platform built on the full-text, vector, and geospatial search capabilities of Apache Lucene"
